.travis.yml

```
@@ -1,39 +1,23 @@
-  "script": [
-    "./flutter/bin/flutter test"
-  ],
-  "deploy": [
-    {
-      "provider": "pages",
-      "strategy": "git",
-      "on": {
-        "branch": [
-          "drewdev"
-        ]
+  "addons": { "apt": { "packages": ["lib32stdc++6"] } },
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)